junit-test/shared/src/test/scala/scala/scalanative/junit/AsyncTest.scala

```diff
@@ -0,0 +1,31 @@
+package scala.scalanative.junit
+
+// Ported from Scala.js
+
+import org.junit.Test
+
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.Future
+import scala.scalanative.junit.async._
+import scala.scalanative.junit.utils._
+
+class AsyncTest {
+  @Test
+  def success(): Unit = await {
```

Comment:
```suggestion
  def success(): AsyncResult = await {
```
That's the whole point of `junit.async._`.

Also for the two methods below.